Source/Core/Core/WiiRoot.cpp

```diff
@@ -198,6 +199,8 @@ void InitializeWiiRoot(bool use_temporary)
   {
     File::SetUserPath(D_SESSION_WIIROOT_IDX, File::GetUserPath(D_WIIROOT_IDX));
   }
+
+  s_wii_root_initialized = true;
```

Comment:
Should we add an assertion that !s_wii_root_initialized in InitializeWiiRoot?

Reply:
Done.